modules/samples/product/Chad/main_www/js/WSRequest.js

```diff
@@ -350,10 +350,10 @@ WSRequest.util = {
                     var xmlSerializer = document.implementation.createLSSerializer();
                     return xmlSerializer.writeToString(payload);
                     break;
-                case "safari":
-                // use the safari method
-                    window.alert("This browser is not supported in this sample. Please use a compatible browser like Firefox, Opera, Internet Explorer");
-                    throw new WebServiceError("Not implemented", "WSRequest.util._serializeToString doesn't support Safari browser.");
+                case "webkit":
+                	console.log("sdffds");
```

Comment:
Could you please explain the meaning of this log? If it's not needed, can you remove it?

console.log("sdffds");